search/domain/base.py

```diff
@@ -83,11 +83,8 @@ class SchemaBase(Base):
 
     def __getattr__(self, key: str) -> Any:
         """Get a schema attribute."""
-        try:
-            super(SchemaBase, self).__getattr__(key)
```

Comment:
Turns out ``dict`` has no ``__getattr__`` (of course...)

Reply:
That's so strange.